placement-rules-in-sql.md

```diff
@@ -5,276 +5,363 @@ summary: Learn how to schedule placement of tables and partitions using SQL stat
 
-The detailed user scenarios are as follows:
+## Overview
 
-- Merge multiple databases of different applications to reduce the cost on database maintenance
-- Increase replica count for important data to improve the application availability and data reliability
-- Store new data into NVMe storage and store old data into SSDs to lower the cost on data archiving and storage
-- Schedule the leaders of hotspot data to high-performance TiKV instances
-- Separate cold data to lower-cost storage mediums to improve cost efficiency
-- Support the physical isolation of computing resources between different users, which meets the isolation requirements of different users in a cluster, and the isolation requirements of CPU, I/O, memory, and other resources with different mixed loads
+With the Placement Rules in SQL feature, you can [create placement policies](#create-and-attach-placement-policies) and configure desired placement policies for data at different levels, with granularity from coarse to fine as follows:
 
-## Specify placement rules
+| Level            | Description                                                                          |
+|------------------|--------------------------------------------------------------------------------------|
+| Cluster          | By default, TiDB configures a policy of 3 replicas for a cluster. You can configure a global placement policy for your cluster. For more information, see [Cluster configuration](#specify-the-number-of-replicas-globally-for-a-cluster). |
+| Database         | You can configure a placement policy for a specific database. For more information, see [Specify a default placement policy for a database](#specify-a-default-placement-policy-for-a-database). |
+| Table            | You can configure a placement policy for a specific table. For more information, see [Specify a placement policy for a table](#specify-a-placement-policy-for-a-table). |
+| Partition        | You can create partitions for different rows in a table and configure placement policies for partitions separately. For more information, see [Specify a placement policy for a partitioned table](#specify-a-placement-policy-for-a-partitioned-table). |
 
-To specify placement rules, first create a placement policy using [`CREATE PLACEMENT POLICY`](/sql-statements/sql-statement-create-placement-policy.md):
+> **Tip:**
+>
+> The implementation of *Placement Rules in SQL* relies on the *placement rules feature* of PD. For details, refer to [Configure Placement Rules](https://docs.pingcap.com/zh/tidb/stable/configure-placement-rules). In the context of Placement Rules in SQL, *placement rules* might refer to *placement policies* attached to other objects, or to rules that are sent from TiDB to PD.
 
-```sql
-CREATE PLACEMENT POLICY myplacementpolicy PRIMARY_REGION="us-east-1" REGIONS="us-east-1,us-west-1";
-```
+## Constraints
```

Comment:
```suggestion
## Limitations
```
How about `limitations`, It's a little confusing with the `### CONSTRAINTS formats` part.